src/main/java/co/cask/hydrator/salesforce/SObjectsDescribeResult.java

```diff
@@ -88,17 +87,19 @@ public SObjectsDescribeResult(PartnerConnection connection, Collection<String> s
    */
   public Field getField(String sObjectName, String fieldName) {
     Map<String, Field> fields = objectToFieldMap.get(sObjectName.toLowerCase());
-    return fields == null ? null : fields.get(fieldName);
+    return fields == null ? null : fields.get(fieldName.toLowerCase());
```

Comment:
you still miss toLowerCase transformation in constructor, don't you?

Reply:
I have added it in stream creation above that code.

(field) -> field.getName().toLowerCase();